CMakeLists.txt

```diff
@@ -49,6 +49,11 @@ else()
   find_package(bout++ REQUIRED)
 endif()
 
+
+find_package(MPI REQUIRED)
+find_package(NESO-Particles REQUIRED)
```

Comment:
We provide a `find_sycl_if_required` function in NP that is probably being called via Reactions? You could call it here for robustness.